packages/gdl-frontend/components/Reader/Toolbar.js

```diff
@@ -18,50 +24,60 @@ import SrOnly from '../SrOnly';
 import { colors } from '../../style/theme';
 import media from '../../style/media';
 import Favorite, { FavoriteIcon } from '../Favorite';
-import { flexCenter } from '../../style/flex';
 
 type Props = {
   book: BookDetails,
   onRequestClose(): void,
   userHasEditAccess?: boolean,
-  chapter: { id: number, seqNo: number }
+  chapter: { id: number, seqNo: number },
+  changeFontSize: (newFontSize: number) => void,
+  fontSize: number
 };
 
 const Toolbar = ({
   book,
   chapter,
   userHasEditAccess,
-  onRequestClose
+  onRequestClose,
+  changeFontSize,
+  fontSize
 }: Props) => (
-  <Div>
-    {/* Create single string for page / of x. Reads better in screen readers. Otherwise each thing is on a new line */}
-    <div>{`${chapter.seqNo} / ${book.chapters.length}`}</div>
-    <Buttons>
-      {userHasEditAccess && (
-        <Link
-          href={{
-            pathname: '/admin/edit/book',
-            query: {
-              id: book.id,
-              lang: book.language.code,
-              chapterId: chapter.id
-            }
-          }}
-        >
-          <IconButton title="Edit book">
-            <EditIcon />
-          </IconButton>
-        </Link>
-      )}
-      <FavButton book={book} />
-      <IconButton onClick={onRequestClose}>
-        <CloseIcon />
-        <SrOnly>
-          <Trans>Close book</Trans>
-        </SrOnly>
-      </IconButton>
-    </Buttons>
-  </Div>
+  <Grid container className={styledGrid}>
+    <Grid item>
```

Comment:
The `Grid` introduce a layer with margin. Fix would be `style={{ alignSelf: 'flex-end' }}`